JavaScript/packages/resource-generator/lib/code-writers.ts

```diff
@@ -163,6 +176,9 @@ export function GenerateCode(root: any): CodeWriter[] {
         else if (token instanceof Array) {
             lines.push(new ArrayWriter(tokenName, token));
         }
+        else if (typeof token === "boolean") {
```

Comment:
Didn't work with instanceof Boolean

Reply:
Yes, in javascript `true` is a boolean **primitive**, only boolean objects are instances of Boolean.